unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/RPCSocialApiBridge.cs

```diff
@@ -396,6 +457,13 @@ private void LogIncomingFriendshipUpdateEventError(SubscribeFriendshipEventsUpda
             {
                 cancellationToken.ThrowIfCancellationRequested();
 
+                if (socialClient == null) { await WaitForSocialClient(cancellationToken); }
+                if (socialClient == null)
+                {
+                    Debug.LogError("Getting all friends failed due to lack of client");
+                    return new FriendshipEventResponse();
```

Comment:
i would rather prefer to throw an exception than returning an invalid value so the inner layers can react accordingly

Reply:
This is OOS, will review in the future